sonar-scm-git-plugin/src/main/java/org/sonarsource/scm/git/GitScmProviderBefore77.java

```diff
@@ -199,12 +200,15 @@ private Ref resolveTargetRef(String targetBranchName, Repository repo) throws IO
   }
 
   @CheckForNull
-  private static Ref getFirstExistingRef(Repository repo, String first, String second) throws IOException {
-    Ref targetRef = repo.exactRef(first);
-    if (targetRef != null) {
-      return targetRef;
-    }
-    return repo.exactRef(second);
+  private static Ref getFirstExistingRef(Repository repo, String... refs) throws IOException {
+	Ref targetRef = null;
+	for (String ref : refs) {
+		targetRef = repo.exactRef(ref);
```

Comment:
Wrong format. Please use the format rules here: https://github.com/SonarSource/sonar-developer-toolset
I can fix it for you if you'd prefer.